### PR TITLE
feat: support steel path fissure filtering

### LINKED ISSUE #864: :rocket: Fissure panel: Show if Steel Path Fissure or not
### What would you like to see happen?

With the Warframe Update 32 Veilbreaker came Steel Path mode in Fissures. It would be cool if the Fissure panel in warframe hub would display if a node is Steel Path or not. Right now I don't think you can see the difference from the UI. Thanks in advance.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by TobiTenno**
**Summary:**
Steel path filtering

---
**Fixes issue (include link):**
closes #864

### CODE REVIEW

File: static/lang/en.json [hunk 1]
```diff
@@ -93,7 +93,8 @@
     "header": "Fissures",
     "voidstorm": "Void Storm",
     "choice": "Choose which fissures to show",
-    "both": "Both"
+    "both": "All",
```
Comment (EricSihaoLin) [inline static/lang/en.json]: nit picky, but perhaps change the key as well?

**Review by EricSihaoLin — DISMISSED** [review]

File: static/lang/en.json (hunk 1 above)
Comment (TobiTenno) [reply static/lang/en.json]: sure
```suggestion
    "all": "All",
```

File: components/panels/FissuresPanel.vue [hunk 2]
```diff
@@ -71,14 +80,46 @@ fissureIcons.push(lith, meso, neo, axi, requiem);
...
+        display: 'inline',
+        'vertical-align': 'middle',
+      },
+      archwing,
+      steelPath,
+      options: [
+        { text: this.$t('fissures.header'), value: 'fissures' },
+        { text: this.$t('fissures.voidstorm'), value: 'storms' },
+        { text: this.$t('fissures.steelPath'), value: 'steelPath' },
+        { text: this.$t('fissures.both'), value: 'fissures-storms-steelPath' },
```
Comment (TobiTenno) [inline components/panels/FissuresPanel.vue]:
```suggestion
        { text: this.$t('fissures.all'), value: 'fissures-storms-steelPath' },
```

**Review by EricSihaoLin — APPROVED** [review]

**Pull request merged**